dlp/snippets/deid.py

```diff
@@ -937,6 +937,80 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
+    """Uses the Data Loss Prevention API to deidentify sensitive data in a
```

Comment:
de-identify (when in comments)

Reply:
Thank you for reviewing; updated in the next push. Please check